src/main/java/net/imagej/ops/threshold/ThresholdNamespace.java

```diff
@@ -122,6 +126,24 @@
 		return result;
 	}
 
+	// -- binary mask --
+
+	@OpMethod(op = net.imagej.ops.threshold.binaryMask.DefaultBinaryMask.class)
```

Comment:
@dietzc I haven't had time to create a PR from it. I find the idea of having an upper and a lower threshold pretty intriguing! I'd like to wait for a clean up of `ThresholdNamespace` till the discussion on nested namespaces has progressed (after talking with @ctrueden about that). So I'd go ahead and merge this and we might get back to the idea while cleaning up?